### DISCUSSION TITLE
v1.1.0 alpha

### DISCUSSION THREAD

**Discussion by Yalz** [Informatievlaanderen/VSDS-LDESServer4J#758] [Announcements]
## What's Changed

* fix: Resolved OpenAPI errors due to JENA models #676 
* fix: Resolved multi level fragmentation regression #664 
* fix: Use POST for creating Event Streams (previously PUT) #673 
* fix(docs): Updated documentation to use server v1 spec #703 
* fix: Improved error handling on invalid API requests #632 #715 
* fix: Server no longer returns an error when timestampPath or VersionOfPath is missing when creating a new eventstream #724 

* feat: Mongock migration from server v0 to server v1
* feat: Improved performance for deleting collections and views
* feat: Retention policies can now be combined
* feat: Enabled SHACL validation for view creation

### Version Bumps:

* build(deps): bump impsort-maven-plugin from 1.8.0 to 1.9.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/603
* build(deps): bump maven-assembly-plugin from 3.5.0 to 3.6.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/615
* build(deps): bump mongock-bom from 5.2.4 to 5.3.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/616
* build(deps): bump spring-boot.version from 3.0.6 to 3.1.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/622
* build(deps): bump maven-source-plugin from 3.2.1 to 3.3.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/624
* build(deps-dev): bump mockito-core from 5.2.0 to 5.3.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/502
* build(deps): bump maven-surefire-plugin.version from 3.0.0 to 3.1.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/595
* build(deps): bump junit-platform-suite from 1.9.2 to 1.9.3 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/552
* build(deps): bump junit-jupiter from 5.9.2 to 5.9.3 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/551
* build(deps): bump apache-jena.version from 4.7.0 to 4.8.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/546
* build(deps): bump jacoco-maven-plugin from 0.8.9 to 0.8.10 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/547
* build(deps): bump spring-boot.version from 3.0.5 to 3.0.6 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/534
* build(deps-dev): bump cucumber.version from 7.11.2 to 7.12.0 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/559
* build(deps-dev): bump mockito-core from 5.3.0 to 5.3.1 by @dependabot in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/540

**Full Changelog**: https://github.com/Informatievlaanderen/VSDS-LDESServer4J/compare/0.0.7...v1.1.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/Informatievlaanderen/VSDS-LDESServer4J/releases/tag/v1.1.0-alpha'>v1.1.0 alpha</a>.</em>

**Comment by xdxxxdx** [comment]
Super cool this kind of release note !
